docs/source/model_design/risk_factors.rst

```diff
@@ -38,11 +38,11 @@ What is a risk factor?
   Update this section once this page has been filled in further.
 
 .. todo::
-  
```

Comment:
Protip for other reviewers.  Click the gear icon at the top and select the box to hide whitespace changes.  Nathaniel uses a good editor that strips erroneous whitespace by default and so his commits have a bunch of changes that aren't content changes. 
![image](https://user-images.githubusercontent.com/6313865/76140047-5a0e1300-600b-11ea-8338-4f872c773f80.png)